pipeline/transformer/transformer_calm/src/test/scala/weco/pipeline/transformer/calm/transformers/CalmMergeCandidatesTest.scala

```diff
@@ -0,0 +1,90 @@
+package weco.pipeline.transformer.calm.transformers
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import weco.catalogue.internal_model.identifiers.{
+  IdState,
+  IdentifierType,
+  SourceIdentifier
+}
+import weco.catalogue.internal_model.work.MergeCandidate
+import weco.catalogue.source_model.generators.CalmRecordGenerators
+
+class CalmMergeCandidatesTest
+    extends AnyFunSpec
+    with Matchers
+    with CalmRecordGenerators {
+
+  it("creates a Sierra mergeCandidate from the BNumber field") {
+    val bnumber = "b1234567"
+    val record = createCalmRecordWith(
+      "BNumber" -> bnumber
+    )
+    val mergeCandidates = CalmMergeCandidates(record)
+
+    mergeCandidates should contain only MergeCandidate(
+      IdState.Identifiable(
+        SourceIdentifier(
+          identifierType = IdentifierType.SierraSystemNumber,
+          ontologyType = "Work",
+          value = bnumber
+        )
+      )
+    )
+  }
+
+  it("creates Miro mergeCandidates from Wheels fields") {
+    val miroIds = List("M0000001", "M0000002", "M0000003")
+    val record = createCalmRecordWith(
+      miroIds.map(id => "Wheels" -> id): _*
+    )
+    val mergeCandidates = CalmMergeCandidates(record)
+
+    mergeCandidates should contain allElementsOf miroIds.map(
```

Comment:
I would expect:

```suggestion
    mergeCandidates shouldBe miroIds.map(
```

to work. What am I missing?

Reply:
Strictly, it's that I don't mind whether ordering is preserved